jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -1299,6 +1295,10 @@ BoundMethodHandle rebind() {
 	}
 	
 /*[IF Sidecar19-SE-OpenJ9]*/
+	protected MethodHandle getTarget() {
```

Comment:
Does this need to be protected or can it be package private?

Reply:
`protected` seems needed by `DelegatingMethodHandle`
```
abstract class java.lang.invoke.DelegatingMethodHandle extends java.lang.invoke.MethodHandle {
  protected abstract java.lang.invoke.MethodHandle getTarget();
}
```

Reply:
a subclass can increase the visibility of a method from package to protected.  As DelegatingMH isn't API, it's free to have a protected method.

Reply:
Agreed. `getTarget()` can be made to package private which can pass signature tests for `java.lang.invoke.MethodHandle`.

Looked again the initial compilation failure due to removal of this method as following:
```
/tmp/bld_369599/openjdk/jdk/src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java:510: error: method does not override or implement a method from a supertype
        @Override
```
But `MethodHandleImpl` doesn't inherit from `MethodHandle`. 
```
abstract class java.lang.invoke.MethodHandleImpl {
}
```
It is the inner classes of `MethodHandleImpl` that attempt to override `getTarget()`
```
Compiled from "MethodHandleImpl.java"
final class java.lang.invoke.MethodHandleImpl$AsVarargsCollector extends java.lang.invoke.DelegatingMethodHandle {
protected java.lang.invoke.MethodHandle getTarget();
}
....
final class java.lang.invoke.MethodHandleImpl$WrappedMember extends java.lang.invoke.DelegatingMethodHandle {
protected java.lang.invoke.MethodHandle getTarget();
}
```
I just realized that J9 has a stub class `DelegatingMethodHandle` which doesn't have `getTarget()`, and inherits it from `MethodHandle` instead.